src/server_manager/web_app/shadowbox_server.ts

```diff
@@ -73,37 +80,30 @@ export class ShadowboxServer implements server.Server {
 
   async addAccessKey(): Promise<server.AccessKey> {
     console.info('Adding access key');
-    return makeAccessKeyModel(
-      await this.apiRequest<AccessKeyJson>('access-keys', {method: 'POST'})
-    );
+    return makeAccessKeyModel(await this.apiRequest<AccessKeyJson>('access-keys', 'POST'));
   }
 
   renameAccessKey(accessKeyId: server.AccessKeyId, name: string): Promise<void> {
     console.info('Renaming access key');
-    const body = new FormData();
-    body.append('name', name);
-    return this.apiRequest<void>('access-keys/' + accessKeyId + '/name', {method: 'PUT', body});
+    return this.apiRequest<void>('access-keys/' + accessKeyId + '/name', 'PUT', {
```

Comment:
See my comment on apiRequest. This could be `this.api.request<void>('access-keys/' + accessKeyId + '/name', 'PUT', {body: form_as_string})`

We may want a function that goes from a dict to a form-encoded string.